app/views/state.js

```diff
@@ -320,6 +321,7 @@ YUI.add('juju-app-state', function(Y) {
       var url = req.path,
           query = req.query,
           state = {};
+      url = url.replace(this.get('baseUrl'), '');
```

Comment:
It's trivial but can you add a comment here plz - I really want to keep this as documented as possible :)